### PR TITLE
#8 - Fix entities links (Backend)

### LINKED ISSUE #8: Fix entities links (Backend)

### PR DESCRIPTION
**Pull request by ImmortalIdiot**

### CODE REVIEW

File: backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Direction.java [hunk 1]
```diff
@@ -3,13 +3,17 @@
...
+import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import java.util.List;
 
 @Entity
 @Table(name = "directions")
 @Data
+@AllArgsConstructor
```
Comment (NN171) [inline backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Direction.java]: RequiredArgsConstructor, which included in Data annotation is enough, so you can remove it

File: backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Direction.java [hunk 2]
```diff
@@ -3,13 +3,17 @@
...
+@NoArgsConstructor
```
Comment (NN171) [inline backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Direction.java]: Similar to the previous

**Review by NN171 — COMMENTED** [review]
That's okay

File: backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Direction.java (hunk 2 above)
Comment (ImmortalIdiot) [reply backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Direction.java]: Removed

File: backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Direction.java (hunk 1 above)
Comment (ImmortalIdiot) [reply backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Direction.java]: Removed

**Review by NN171 — APPROVED** [review]
Okay

**Pull request merged**